tests/test_loss.py

```diff
@@ -81,3 +81,55 @@ def test_soft_cross_entropy(self, tensors, use_weights, reduction):
 
         torch.testing.assert_close(loss_indices, loss_onehot)
         torch.testing.assert_close(loss_indices, loss_reference)
+
+
+class TestDiceLoss:
+
+    @pytest.fixture()
+    def target(self):
+        target = torch.tensor([[[-1], [0], [1], [2]]]).repeat(1, 1, 4)
+        target = torch.cat([target, target[:, [1, 2, 3, 0]], target[:, [2, 3, 0, 1]], target[:, [3, 0, 1, 2]]], dim=0)
+        return target
+
+    @pytest.fixture()
+    def correct_input(self, target):
+        input = target.clone()
+        input[input == -1] = 1  # replace negative label with class prediction
+        input = F.one_hot(input)
+        input = torch.movedim(input, 3, 1)
+        return input
+
+    @pytest.fixture()
+    def incorrect_input(self, correct_input):
+        return correct_input[[3, 2, 1, 0]]
+
+    @pytest.mark.filterwarnings('ignore::UserWarning')
+    @pytest.mark.parametrize('squared_pred', [True, False])
```

Comment:
These parameterize that are shared can be put on the class itself:
```
@pytest.mkar.parametrize(..)
@pytest.mark.parametrize(..)
class TestDiceLoss:
    ...
```

Reply:
Awesome, done! thanks 🙂